src/components/Metrics/Metrics-i18n.json

```diff
@@ -20,6 +20,9 @@
     "topic_metrics_help_text": "Bytes incoming and outgoing are the total bytes for all topics or total bytes for a selected topic in the Kafka instance. This metric enables you to assess data transfer in and out of your Kafka instance. To modify incoming and outgoing bytes, you can adjust topic message size or other topic properties as needed.",
     "topic_partition_size_popover_header": "Topic partition size chart",
     "topic_partition_size_help_text": "Topic partition size is the log size for each partition in a selected topic. This metric enables you to assess the amount of data used in one or more topic partitions. To reduce the topic partition size, you can decrease the retention time or the retention size, or modify the cleanup policy as needed. You can also increase the number of partitions for the topic.",
+    "topic_incoming_message_rate": "Incoming message rate",
+    "topic_incoming_message_rate_popover_header": "Incoming messages chart",
+    "topic_incoming_message_rate_help_text": "Incoming message rate is the number of messages per second received by one or more topics in the Kafka instance over time. This metric enables you to verify that messages are produced to topics and that producers are functioning properly. To modify the incoming messages per topic, you can adjust the corresponding producer as needed.",
```

Comment:
Remember to also set the correct y axis label, in the design I see "Messsages/second" is used

Reply:
Yep, I have add the current y-axis label https://github.com/redhat-developer/app-services-ui-components/blob/2688536f9388913ac364abf80a5e8f1d174573e0/src/components/Metrics/components/ChartIncomingMessage.tsx#L97